### ISSUE TITLE
Add ros2 compatibility on ros2 branch

### ISSUE THREAD

**Issue by dmiklic** [romb-technologies/ariac_environment#4]
Use the appropriate branch of the [p2os repo](https://github.com/allenh1/p2os) (ros2 or ros2_port)

**Comment by juricapapak** [comment]
During the process of building the ariac_environment package on ros2 crystal I'm getting the following error:
```
CMake Error at CMakeLists.txt:7 (find_package):
  By not providing "Findcatkin.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "catkin", but
  CMake did not find one.

  Could not find a package configuration file provided by "catkin" with any
  of the following names:

    catkinConfig.cmake
    catkin-config.cmake

  Add the installation prefix of "catkin" to CMAKE_PREFIX_PATH or set
  "catkin_DIR" to a directory containing one of the above files.  If "catkin"
  provides a separate development package or SDK, be sure it has been
  installed.
```

I've successfully installed all the dependencies using _rosdep_  and have catkin installed, just not through ros2. I don't see any ros2 branches on the official [repo](https://github.com/ros/catkin/). I've had similar issues before but I've always been able to find the appropriate .cmake config file. Not this time, presumably because it's not a ros2 package.

**Comment by dmiklic** [comment]
Yes, ROS2 does not use Catkin any more. So instead of Catkin (build system) and catkin_make (build tool), it uses ament as the build system and colcon as the build tool. Please check the [Migration guide](https://index.ros.org/doc/ros2/Contributing/Migration-Guide/) for more details.

**Comment by dmiklic** [comment]
Also, check out [this](http://gazebosim.org/tutorials?tut=ros2_overview) and [this](https://github.com/ros-simulation/gazebo_ros_pkgs/issues/512) for some Gazebo specifics in ROS2.

**Comment by dmiklic** [comment]
However, looking at the status of the [p2os port](https://github.com/allenh1/p2os/compare/ros2_port) we might have to start thinking of a backup plan, maybe using [ros2 bridge](https://github.com/ros2/ros1_bridge).
